### ISSUE TITLE
Projects with MIDI input freeze on load when any MIDI input is turned on

### ISSUE THREAD

**Issue by zralok123** [nomelif/Audionodes#38]
I use the latest Audionodes with Blender version 3.0. When I turn on the keyboard (which I use as the MIDI input) or use a MIDI driver, Blender freezes as soon as I load any project containing the MIDI input node. I'm sure that the driver of the keyboard is not problematic, since it does exactly the same thing with any other MIDI input. However, when I load the same project while no MIDI input is present in the computer, the project loads perfectly. Then, when I configure the MIDI input and assign the keyboard to it, the synthesizer made with Audionodes works in the way it should. So the only way I can use Audionodes with MIDI is:

1. Launching Blender first
2. Loading the project with the synthesizer
3. Turning on the keyboard and connecting it to the computer
4. Selecting the keyboard in the MIDI input node

![conf_failed](https://user-images.githubusercontent.com/95490181/146797919-e2241ca2-33b1-40b8-9139-08f2d25eb199.png)
![configuration1](https://user-images.githubusercontent.com/95490181/146798604-b86a6365-34ac-445b-bbbc-50e2c9ad3e38.png)
![configuration2](https://user-images.githubusercontent.com/95490181/146798610-a4c8c76e-fcae-44e7-960d-9579456ec593.png)

**Comment by ollpu** [comment]
Indeed, something racy is going on. I'm having a hard time observing the freeze if I try to print/log something before it happens.

This is the best stack trace I could get for now:
![image](https://user-images.githubusercontent.com/7241014/160496826-c61f523f-e8e4-48c2-9d4f-4e620399b19c.png)

It's likely around [here](https://github.com/FluidSynth/fluidsynth/blob/master/src/drivers/fluid_winmidi.c#L583) because the SYSEX thread is already running.

Unfortunately I don't know if I will have time to look deeper in the near future.
